manifest.json

```diff
@@ -41,5 +40,8 @@
       "16": "favicon-16.png"
     },
     "default_title": "Notist"
+  },
+  "externally_connectable": {
+    "matches": ["http://notist.io/*"]
```

Comment:
Would it still work if you did `"matches": ["http://notist.io/explore"]` instead